widget/entry.go

```diff
@@ -910,6 +910,7 @@ func (e *Entry) pasteFromClipboard(clipboard fyne.Clipboard) {
 
 	e.updateTextAndRefresh(provider.String())
 	e.CursorRow, e.CursorColumn = e.rowColFromTextPos(pos + len(runes))
```

Comment:
It looks like we're refreshing at least twice now per paste (3 times if e.selecting)? Is there a way to refactor this down to one refresh (or max 2)? Or does it not really matter since they're all but guaranteed to be on the same frame anyway?

Reply:
I looked at that and it would mean introducing 2 new methods, breaking down the content of the two above.
I figured that given this is on a paste operation it wasn't likely a big deal...

Reply:
Sounds good, maybe let's create a ticket tagged with optimization for reducing Entry refreshes? probably not high priority at all but I think something that should eventually be looked into

Reply:
Done :)